dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -62,7 +62,7 @@ bool bitmapNonEmpty(const OpenDDS::RTPS::SequenceNumberSet& snSet)
       }
       for (int bit = 31; bit >= 0; --bit) {
```

Comment:
This should probably just create a mask and do a single comparison rather than loop over individual bits.

Reply:
I'll note it for later.